addons/udes_stock/tests/__init__.py

```diff
@@ -33,3 +33,4 @@
 from . import test_stock_location_picking_zone
 from . import test_location_countable
 from . import test_location_state
+from . import test_product_template  # noqa: F401
```

Comment:
don't all of the other `from . import ...` get imported but unused? is there a reason it's only on test_product_template?

Reply:
I only changed (added) that line; changing all the others seems like a lot of noise, especially as we don't have a consistent linting policy atm.